src/index.js

```diff
@@ -1,71 +1,75 @@
-let AWS = require('aws-sdk');
-let Campus = require('./modules/campus');
-let College = require('./modules/college');
-let Course = require('./modules/course');
-let Curriculum = require('./modules/curriculum');
-let Department = require('./modules/department');
-let Enrollment = require('./modules/enrollment');
-let fs = require('fs');
-let Major = require('./modules/major');
-let MicroCache = require('micro-cache');
-let Person = require('./modules/person');
-let Program = require('./modules/program');
-let Registration = require('./modules/registration');
-let Section = require('./modules/section');
-let Term = require('./modules/term');
-let TestScore = require('./modules/testscore');
-let winston = require('winston');
-
-let FileCertificate = {
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const MicroCache = require('micro-cache');
+const log4js = require('@log4js-node/log4js-api');
+const _ = require('lodash');
+const Campus = require('./modules/campus');
+const College = require('./modules/college');
+const Course = require('./modules/course');
+const Curriculum = require('./modules/curriculum');
+const Department = require('./modules/department');
+const Enrollment = require('./modules/enrollment');
+const Major = require('./modules/major');
+const Person = require('./modules/person');
+const Program = require('./modules/program');
+const Registration = require('./modules/registration');
+const Section = require('./modules/section');
+const Term = require('./modules/term');
+const TestScore = require('./modules/testscore');
+
+const FileCertificate = {
   readCertificate: async (opts) => {
-    if (opts.cert === '' || opts.key === '' ||
-      !fs.existsSync(opts.cert) || !fs.existsSync(opts.key)) {
+    if (opts.cert === '' || opts.key === ''
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
```

Comment:
Is it possible to disable this for the entire file since there are several lines that need to be disabled?

Reply:
I don't like to do that for security warnings so that each one has to be evaluated when you see it. Otherwise you may add a security hole and not know.